helm-charts/splunk-otel-collector/values.yaml

```diff
@@ -106,6 +106,18 @@ splunkPlatform:
     #   requests_per_second is the average number of requests per seconds.
     queueSize: 5000
 
+    # When enabled, it uses file_storage extension to persist the queue data.
+    # This can be used to prevent data-loss during restart
+    # NOTE: The File Storage extension will persist state to the node's local file system.
+    # While using the persistent queue it is advised to increase memory limit for agent (agent.resources.limits.memory)
+    # to 1Gi.
+    persistentQueueEnabled:
```

Comment:
- We should move these configurations under sendingQueue since they are not different queues.
- We probably need to note the [alpha](https://github.com/open-telemetry/opentelemetry-collector/blob/main/exporter/exporterhelper/README.md#persistent-queue) status of this component.

Reply:
As I checked the values.yaml file, `persistentQueueEnabled` is already moved under `sendingQueue `.

Reply:
My mistake.